### ISSUE TITLE
Questions of paper "PreRoutGNN"

### ISSUE THREAD

**Issue by young03600** [Thinklab-SJTU/EDA-AI#33]
Hi,
I found the fascinating paper "PreRoutGNN for Timing Prediction with Order Preserving Partition: Global Circuit Pre-training, Local Delay Learning and Attentional Cell Modeling".

- As mentioned in the paper, the features will be listed in Appendix, but I couldn't find the table for features.

- Question about the model for auto-encoder. Does the auto-encoder only contain linear layers?

- It's appreciated that the code will be published. I found the code hasn't be published in this repo. When will the code be published?

Thanks.

**Comment by zerzerzerz** [comment]
### Node and Edge Features
- Thank you for your suggestion. We will modify the appendix in the paper to include the features of the nodes and edges. We list the node and edge features below.
- Node features

|name|dim|
|-|-|
|is primary I/O or not|1
|is fanin or fanout|1
|distance to the 4 die area boundaries|4
|pin capacitance|4
|node level|1
|node level encoding|$2 \times L$
|node and graph feature from graph-encoder|$2 \times d$


- Edge features
  - We use the same edge features as the [TimingGCN](https://github.com/TimingPredict/TimingPredict)


### Structure of Graph Encoder
  - We use the graph structure Deep GCNII in file `PreRoutGNN/model/deep_gcnii.py`
  - The function to generate edge message and aggregate node feature is implemented with MLP.

### Coda Availability
- We have already make our code publicly available in `PreRoutGNN/README.md`


Thank you for your interest in our work. We are looking forward to your feedback.
